### PR TITLE
Feature ests 136991

### PR DESCRIPTION
**Pull request by krp2**
- [ ] I have read the [CONTRIBUTION GUIDE][contributing]

- [ ] This is a bug

- [ ] This is a feature request

- [ ] This is a syntax/style fix

- [ ] This is an improvement to code quality

---
**Please describe your changes below:**
(Note: A clear description of changes is required to approve any pull requests)

_"Changed the implementation for verification because..."_

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,202 +1,253 @@
...
+						<phase>validate</phase>
+						<goals>
+							<goal>unpack</goal>
+						</goals>
+						<configuration>
+							<artifactItems>
+								<artifactItem>
+									<groupId>com.dell.cpsd</groupId>
+									<artifactId>amqp-contract-extension</artifactId>
+									<includes>message/**/*.*</includes>
```
Comment (ganapn) [inline pom.xml]: Same comment as other repo. messaging instead of message?

File: pom.xml [hunk 2]
```diff
@@ -1,202 +1,253 @@
...
+									<outputDirectory>${project.basedir}/target/main/resources/sample-service-api/schema/json</outputDirectory>
+								</artifactItem>
```
Comment (ganapn) [inline pom.xml]: project.builddir can be used instead of project.basedir/target correct?

File: pom.xml (hunk 1 above)
Comment (krp2) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (krp2) [reply pom.xml]: Yes. done

**Comment by krp2** [comment]
Incorporated  review comments

File: src/test/java/com/dell/cpsd/sample/service/ValidateExamplesTest.java [hunk 3]
```diff
@@ -22,13 +22,14 @@
 {
     public static final String SCHEMA_DIR   = "/sample-service-api/schema/json/";
     public static final String EXAMPLE_DIR  = "/sample-service-api/schema/example/";
+    public static final String EXTERNAL_SCEMA_DIR  = "messaging/properties";
```
Comment (ganapn) [inline src/test/java/com/dell/cpsd/sample/service/ValidateExamplesTest.java]: Correct typo. It should be SCHEMA

Comment (krp2) [reply src/test/java/com/dell/cpsd/sample/service/ValidateExamplesTest.java]: done

**Pull request merged**